packages/rum-core/src/domain/contexts/foregroundContexts.ts

```diff
@@ -65,19 +65,13 @@ export function closeForegroundPeriod() {
 }
 
 function trackFocus(onFocusChange: () => void) {
-  return addEventListener(window, DOM_EVENT.FOCUS, (event) => {
-    if (!event.isTrusted) {
-      return
-    }
+  return addEventListener(window, DOM_EVENT.FOCUS, () => {
     onFocusChange()
   })
 }
 
 function trackBlur(onBlurChange: () => void) {
-  return addEventListener(window, DOM_EVENT.BLUR, (event) => {
-    if (!event.isTrusted) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could we also remove this one? https://github.com/DataDog/browser-sdk/blob/7e0d78cd7e98f374427736ff5682def2935e2493/packages/rum/src/domain/record/observers/moveObserver.ts#L55?

Reply:
I merge my branch with prerelease-v5. This code no longer exist in v5 :)

Reply:
it is still there in [prerelease-v5](https://github.com/DataDog/browser-sdk/blob/prerelease-v5/packages/rum/src/domain/record/observers/moveObserver.ts#L55), is it planned to be removed elsewhere?

Reply:
Ho yes, I remember this one. So my reasoning is that we can still have events with `isTrusted: false `if the user uses `__ddIsTrusted` so this check seems still valid. Wdyt?